src/utils/notifications.js

```diff
@@ -0,0 +1,34 @@
+import mitt from 'mitt'
+
+export const ENotification = {
+  showBusyState: 'showBusyState',
+  hideBusyState: 'hideBusyState',
+  showAlert: 'showAlert',
+  showBanner: 'showBanner',
+  hideBanner: 'hideBanner'
+}
+
+export const EOverlayType = {
+  loading: 0,
+  message: 1
+}
```

Comment:
It actually prevents typos, it depends what IDE do you use but it works with autocompletion (I use Webstorm). One benefit is typos prevention, the other is the autocompletion so I don't have to remember what options are available.

<img width="769" alt="Screenshot 2022-06-23 at 11 36 11" src="https://user-images.githubusercontent.com/17565389/175268107-b3a3bc11-6363-4def-b9fb-3a3ba21cc8c0.png">

About Typescript, I agree, I could use strings here instead of numbers, but when we switch to Typescript at some point (and I really think we should someday) it will be much easier to change that const into an enum.

```
export enum EOverlayType {
  loading,
  message
}
```